spec/lib/mcb/editor/courses_editor_spec.rb

```diff
@@ -426,13 +439,107 @@ def run_new_course_wizard(*input_cmds)
           english: 'equivalence_test',
           science: 'not_required',
           age_range: 'secondary',
+          level: 'secondary',
           course_code: '1X2B',
           recruitment_cycle: '2', # the 2nd option should always be the current recruitment cycle
           application_opening_date: "18 October 2018",
           is_send: true
         }
       }
 
+      describe 'selects the correctly leveled subject' do
+        it 'only shows secondary & language subjects if secondary level is selected' do
```

Comment:
can we double check that this is not the case that is possible in real life

ie
`English` as `SecondarySubject` + `English as a secondary language`
For a `English course`

as far as I am aware 

- 1 or 2 `SecondarySubject`
- 1 `SecondarySubject` with a subject name of `Modern Language` + 1 or more of a `Named Language` from `ModernLanguage`